### PR TITLE
docs: add INSTALL.md.

### PR DESCRIPTION
**Pull request by klihub**

### CODE REVIEW

File: docs/INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+## Post-Install Configuration
+
+The provided packages install `systemd` service files and sample configuration.
+The easiest way to get up and running is to copy the sample configuration in
+the correct place and start CRI Resource Manager using systemd. You can do
+this using the following commands:
+
+```
+cp /usr/share/doc/cri-resource-manager/fallback.cfg.sample /etc/cri-resmgr
```
Comment (marquiz) [inline docs/INSTALL.md:73]: Wrong filename(?) Should be
```
cp /usr/share/doc/cri-resource-manager/fallback.cfg.sample /etc/cri-resmgr/fallback.cfg
```
instead??

File: docs/INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  - install them:
+    - for rpm packages: `sudo rpm -Uvh <packages>`
+    - for deb packages: `sudo dpkg -i <packages>`
+
+## Installing From Sources
+
+Although not recommended, you can install CRI Resource Manager from sources:
+
+  - get the sources: `git clone https://github.com/intel/cri-resource-manager`
+  - build and install: `cd cri-resource-manager; make build install`
```
[the remark this answers is not in the document]
Comment (klihub) [reply docs/INSTALL.md:16]: True, I'll update it.

File: docs/INSTALL.md (hunk 1 above)
Comment (klihub) [reply docs/INSTALL.md:73]: Thanks.

**Comment by marquiz** [comment]
lgtm 👍

**Comment by klihub** [comment]
> lgtm

@kad Is this good enough as an initial version ?

**Review by kad — APPROVED** [review]
lgtm

**Pull request merged**